### DISCUSSION TITLE
大哥,为什么LLVM-MSVC_v143_KernelMode  2.8.1 Release 编译我的驱动,还是有页故障? Debug 没有了

### DISCUSSION THREAD

**Discussion by fgmailbox** [NewWorldComingSoon/llvm-msvc-issues#67] [Announcements]
哥,这是为什么呀? 
为什么现在 Release 还是页故障?
附上我的代码?
[CactiWall.zip](https://github.com/NewWorldComingSoon/llvm-msvc-issues/files/11983145/CactiWall.zip)

**Comment by fgmailbox** [comment]
就是刚加载的时候,发生的,在 knew2.h文件里面

**Comment by gmh5225** [comment]
你的代码可能被优化了，试试MSVC是否也是如此，是的话，你可能需要调整下代码写法

**Comment by fgmailbox** [comment]
如果 msvc wdk 编译器 release 版本编译不会搞崩溃编译器, 我也不会用clang-cl编译器编译内核
